test/fn/create-users.spec.js

```diff
@@ -1,66 +1,81 @@
-const api = require('../api-stub');
-const assert = require('chai').assert;
-const createUsers = require('../../src/fn/create-users');
+const api = require("../api-stub");
```

Comment:
This file has been converted to use double quotes everywhere. Our style is single quotes. Can you switch it back, and maybe enable eslint so it'll fail in future?